SubredditService/Sources/SearchDataSource.swift

```diff
@@ -0,0 +1,70 @@
+import CoreAPI
+import Foundation
+import ListingService
+
+public class SearchDataSource {
+    public private(set) var subreddits = [Subreddit]()
+    public var updated: (() -> Void)?
+
+    private var refreshTask: URLSessionTask?
+    private var loadMoreTask: URLSessionTask?
+
+    public var query: String = ""
+
+    public init( ) {
```

Comment:
Seems we don't need this init as it doesn't do anything (See commented out code)

Reply:
This init was added because Xcode gave an error when SearchDataSource was being used in the SubredditsViewController

Reply:
This was the error : Initializer is inaccessable due to 'internal' protection level